shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -325,7 +327,7 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
                         result.setClassName("android.widget.HorizontalScrollView");
                     }
                 } else {
-                    if (shouldSetCollectionInfo(object)) {
+                    if (Build.VERSION.SDK_INT > Build.VERSION_CODES.JELLY_BEAN_MR2 && shouldSetCollectionInfo(object)) {
```

Comment:
Here and elsewhere, can you add comments about what happens if the SDK versions are not met?

Reply:
Added to the inline comments above this block.